### PR TITLE
Add SameSite None to fix embedded version

### PR DESCRIPTION
**Pull request by GSokol**
The embedded version fof the site failed to use cookies, so it generates
new anonymous id. Fixing it.

## Description of the change

> Description here

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/610)

### CODE REVIEW

**Comment by saikumarrs** [comment]
@GSokol Thanks for raising the PR. However, the changes could inadvertently turn out to be a breaking change for some SDK users. 

Instead, you can add support for `SameSite` storage option [here](https://github.com/rudderlabs/rudder-sdk-js/blob/fe3ea03605920368e52c2ef6733799fab0c3687c/analytics.js#L946-L955). We already support an [option](https://www.rudderstack.com/docs/sources/sdks/rudderstack-javascript-sdk/#the-options-parameter) for the `Secure` header.

**Comment by GSokol** [comment]
@saikumarrs, yes, great advice! Thank you, I will do so!

File: utils/storage/cookie.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import cookie from "rudder-component-cookie";
 import defaults from "@ndhoule/defaults";
 import topDomain from "@segment/top-domain";
+import { parse } from "component-url";
```
Comment (saikumarrs) [inline utils/storage/cookie.js:4]: This is unused. Please remove.

**Comment by saikumarrs** [comment]
> @saikumarrs, yes, great advice! Thank you, I will do so!

Thanks. The changes look good to me. I left a comment to address a minor issue.

As a next step, please sign the CLA using this [form](https://rudderlabs.wufoo.com/forms/rudderlabs-contributor-license-agreement).

**Comment by GSokol** [comment]
@saikumarrs done

**Comment by saikumarrs** [comment]
> @saikumarrs done

@GSokol Thanks. We'll absorb this PR and release a new SDK version next week.
Thanks for understanding.

**Comment by saikumarrs** [comment]
@GSokol Thanks for your patience. We are currently swamped and won't be able to release a new version of the SDK this week. However, we've added this to our immediate sprint roadmap and will include it in the next week's release.
CC @MoumitaM @pallabmaiti

**Review by MoumitaM — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**